src/components/screens/votingQueue/editor/editor.js

```diff
@@ -102,6 +102,7 @@ const getVoteStats = votes =>
 const token = tokenMap.LSK.key;
 const txType = 'vote';
 
+/* eslint-disable max-statements */
```

Comment:
Handling a simple form state shouldn't be so complex. we have this problem in Send form too. some of it due to the complicated usage of `useTransactionFeeCalculation`. We should consider rethinking it later on.

Reply:
We can separate it out to yet another hooook 😄

Reply:
LOL